### PR TITLE
fix borderline in `nace_classification` and `isic_classification`

### LINKED ISSUE #163: Borderline col in classification Bridges
My understanding t=is that the Borderline column should flag as FALSE when the sector classification code is perfectly in scope. 

        code   description            Borderline
eg - 3456, auto manufacture, FALSE
      -3457, auto parts,             TRUE 

This is not the case - instead, all FALSE borderline flags are for sectors not in scope.     
Perhaps my understanding is incorrect?

Thanks - George

### PR DESCRIPTION
**Pull request by jdhoffa**
Closes #163

### CODE REVIEW

File: data-raw/isic_classification.csv [hunk 1]
```diff
@@ -55,19 +55,19 @@ A,1,not in scope,FALSE
...
+52,3,coal,FALSE
+520,4,coal,FALSE
+6,2,oil and gas,FALSE
+61,3,oil and gas,FALSE
+610,4,oil and gas,FALSE
+62,3,oil and gas,FALSE
+620,4,oil and gas,FALSE
+7,2,not in scope,FALSE
+71,3,not in scope,FALSE
+710,4,not in scope,FALSE
```
Comment (maurolepore) [inline data-raw/isic_classification.csv:70]: @georgeharris2deg, is this what you expected?
The PR has more changes than the ones I highlight here, but this snippet should give you an idea of the _kind_ of change that @jdhoffa did. Please check.

**Review by maurolepore — APPROVED** [review]
LGTM. It would be great to have the opinion of @georgeharris2deg

**Comment by maurolepore** [comment]
@jdhoffa, did you update the regression references?

**Review by maurolepore — CHANGES_REQUESTED** [review]
I see errors on CI. It looks like we need to update the regression refferences.

File: data-raw/isic_classification.csv (hunk 1 above)
Comment (georgeharris2deg) [reply data-raw/isic_classification.csv:70]: This looks good to me. Thanks!

**Comment by maurolepore** [comment]
Looks like we re good to merge. @jdhoffa ?

**Comment by jdhoffa** [comment]
Just fixing something that I messed up, will reopen in a moment

**Comment by jdhoffa** [comment]
Ok @maurolepore should be good to go now

File: data-raw/data_dictionary/nace_classification.csv [hunk 2]
```diff
@@ -1,6 +1,6 @@
 dataset,column,typeof,definition
-nace_classification,original_code,double,Original NACE code
-nace_classification,code,double,Formatted NACE code removing periods
+nace_classification,original_code,character,Original NACE code
+nace_classification,code,double,Formatted NACE code with periods removed
```
Comment (maurolepore) [inline data-raw/data_dictionary/nace_classification.csv:3]: :+1:

**Review by maurolepore — APPROVED** [review]

**Pull request merged**